Danki2/Assets/Scripts/Actor/ActorExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿public static class ActorExtensions
+{
+    private static string playerTag = "Player";
+    private static string enemyTag = "Enemy";
+    private static string friendlyTag = "Friendly";
+
+    public static bool IsPlayer(this Actor actor)
+    {
+        return actor.tag == playerTag;
+    }
+
+    public static bool IsEnemy(this Actor actor)
+    {
+        return actor.tag == enemyTag;
+    }
+
+    public static bool IsFriendly(this Actor actor)
+    {
+        return actor.tag == friendlyTag;
+    }
+
+    public static bool Opposes(this Actor actor, Actor target)
+    {
+        return actor.IsEnemy() ^ target.IsEnemy();
```

Comment:
If we get rid of Friendly, this can just become `return actor.tag != target.tag;`. If this goes on Actor, it could call through to a method on TagUtils (see comment about having TagUtils). So you'd have:
```c#
public bool Opposes(Actor target)
{
  return TagUtils.Oppose(this, target);
}
```
on Actor and:
```c#
public static bool Oppose(Actor actor, Actor target)
{
  return actor.tag != target.tag;
}
```
on the TagUtils.